slemicro/README.md

```diff
@@ -261,19 +261,22 @@ The previous environment variables can be used but it requires a few less.
 
 ```bash
 $ ./get_kubeconfig.sh
-/var/folders/t_/vwz7m7x14ll3f_6pszsly1k00000gp/T/tmp.GhEi9ys5
+<kubeconfig contents>
+
+$ ./get_kubeconfig.sh -o $KUBECONFIG
+# Will write/overwrite the KUBECONFIG file
 ```
 
 ```bash
 $ ./get_kubeconfig.sh -h
-Usage: ./get_kubeconfig.sh [-f <path/to/variables/file>] [-n <vmname>] [-i <vmip>]
+Usage: ./get_kubeconfig.sh [-f <path/to/variables/file>] [-n <vmname>] [-i <vmip>] [-o <filename>]
 ```
 
 You can use the script in combination with the `create_vm.sh` one as:
 
 ```bash
 $ ./create_vm.sh -f vm-foobar
-$ KUBECONFIG=$(./get_kubeconfig.sh -w -f vm-foobar)
+$ ./get_kubeconfig.sh -w -f vm-foobar -o $KUBECONFIG
```

Comment:
@e-minguez is it a problem that this existing usage pattern will differ?

For me I like to set a consistent KUBECONFIG in my shell environment, then each time I re-run `create_vm.sh` I overwrite the file, which we can still do but without leaking the files to /tmp?

Reply:
It is ok to me!